src/main/java/org/spongepowered/common/effect/sound/SpongeSoundBuilder.java

```diff
@@ -28,22 +28,19 @@
 
 import net.minecraft.util.ResourceLocation;
 import net.minecraft.util.SoundEvent;
+import org.spongepowered.api.CatalogKey;
 import org.spongepowered.api.effect.sound.SoundType;
+import org.spongepowered.common.util.SpongeCatalogBuilder;
 
-public class SpongeSoundBuilder implements SoundType.Builder {
-
-    @Override
-    public SoundType.Builder from(SoundType value) {
-        return this;
-    }
+public class SpongeSoundBuilder extends SpongeCatalogBuilder<SoundType, SoundType.Builder> implements SoundType.Builder {
```

Comment:
I like final things

Reply:
Finalized it and fixed issue where builder wouldn't actually reset because I overwrote `reset()` where it wasn't necessary.